From 8ce2379d09ebd102f9322d47540e5a0cfb0091dc Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:06 -0700
Subject: [PATCH] o_msg: importing 0001-cavium-hook-pci-and-pcie-support.patch

This is an automatic import of patch 0001-cavium-hook-pci-and-pcie-support.patch, no headers were
detected and a default message was constructed
---
 arch/mips/Kconfig                |    1 +
 arch/mips/cavium-octeon/Makefile |    4 ++++
 2 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index e7b85a3..246c4cb 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -623,6 +623,7 @@ config CAVIUM_OCTEON_REFERENCE_BOARD
 	select SYS_SUPPORTS_HIGHMEM
 	select CPU_CAVIUM_OCTEON
 	select SWAP_IO_SPACE
+	select HW_HAS_PCI
 	select ISA
 	select GENERIC_ISA_DMA
 	select ARCH_MAY_HAVE_PC_FDC
diff --git a/arch/mips/cavium-octeon/Makefile b/arch/mips/cavium-octeon/Makefile
index b5b0d93..5128f61 100644
--- a/arch/mips/cavium-octeon/Makefile
+++ b/arch/mips/cavium-octeon/Makefile
@@ -20,6 +20,10 @@ obj-y += console.o
 
 obj-$(CONFIG_SMP)                     += smp.o
 obj-$(CONFIG_CAVIUM_OCTEON_BOOTBUS_COMPACT_FLASH) += ebt3000_cf.o
+obj-$(CONFIG_PCI)                     += pci-common.o
+obj-$(CONFIG_PCI)                     += pci.o pci_chips.o i8259.o
+obj-$(CONFIG_PCI_MSI)                 += msi.o
+obj-$(CONFIG_PCI)                     += pcie.o
 
 clean:
 	rm -f *.o
-- 
1.5.5.1

